handlers/messagebird/handler_test.go

```diff
@@ -21,10 +21,10 @@ var testChannels = []courier.Channel{
 
 const (
 	receiveURL            = "/c/mbd/8eb23e93-5ecb-45ba-b726-3b064e0c56ab/receive"
-	validReceive          = `{"receiver":"18005551515","sender":"188885551515","message":"Test again","date":1690386569,"date_utc":1690418969,"reference":"1","id":"b6aae1b5dfb2427a8f7ea6a717ba31a9","message_id":"3b53c137369242138120d6b0b2122607","recipient":"18005551515","originator":"188885551515","body":"Test 3","createdDatetime":"2023-07-27T00:49:29+00:00","mms":false}`
```

Comment:
@tybritten I think you added this handler and I assume `188885551515` is a mistake (too long to be a US number) and there isn't something weird going on I'm not aware of...